src/test/java/StringClassTest.java

```diff
@@ -0,0 +1,71 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringClassTest {
+
+    @Test
+    @DisplayName("\"1,2\"을 ,로 split 했을 때 1과 2로 잘 분리되는지 확인하는 학습 테스트를 구현한다.")
+    void split_contains() {
+        //given
+        String data = "1,2";
+
+        //when
+        String[] split = data.split(",");
+
+        //then
+        assertThat(split).contains("1");
+        assertThat(split).contains("2");
```

Comment:
```suggestion
        assertThat(split).contains("1", "2");
```

```suggestion
        assertThat(split).contains("1")
            .contains("2");
```

위의 두 방법도 같은 동작입니다.
containsExactly 를 사용하여 검증해보아도 괜찮겠네요.